feature-governance-impl/src/main/java/io/novafoundation/nova/feature_governance_impl/domain/delegation/delegation/create/chooseTrack/RealChooseTrackInteractor.kt

```diff
@@ -49,6 +54,27 @@ class RealChooseTrackInteractor(
         removeVotesSuggestionRepository.disallowShowRemoveVotesSuggestion()
     }
 
+    override fun observeTracksByChain(chainId: ChainId): Flow<ChooseTrackData> = flowOf {
+        val chain = chainRegistry.getChain(chainId)
+        val govType = chain.governance.firstOrNull { it == Chain.Governance.V2 } ?: return@flowOf ChooseTrackData.empty()
```

Comment:
Is this hardcode valid?
Maybe it's better to pass governance type parameter here

Reply:
Havent seen your comment since I've started the review before you left it so I've started a separate discussion: https://github.com/novasamatech/nova-wallet-android/pull/1407#discussion_r1502160644